client/homebrew/pages/basePages/listPage/brewItem/brewItem.less

```diff
@@ -19,6 +19,21 @@
 	break-inside                : avoid;
 	box-shadow                  : 0px 4px 5px 0px #333;
 	background-color            : #cab2802e;
+	.thumbnail  {
+		width: 80px;
+		height: 100px;
```

Comment:
What do you think about making this a square? 80px x 80px?  My guess the average image is closer to a square, but I actually have no idea.